tests/intern.js

```diff
@@ -0,0 +1,50 @@
+
+define( [ "./config-helper" ], function ( configHelper ) {
+	return {
+		proxyPort: 9000,
+		proxyUrl: "http://localhost:9000/",
+		capabilities: {},
+		environments: [
+			{ browserName: "chrome" },
+			{ browserName: "internet explorer", version: [ "11", "10", "9" ] },
+			// TODO: Firefox 35 is broken for Browserstack, fix when they have Selenium 2.46
+			{ browserName: "firefox", version: [ "34" ] }
+		],
+		maxConcurrency: 2,
+		tunnel: "BrowserStackTunnel",
+		loaders: {
+			"host-node": "requirejs",
+			"host-browser": "external/requirejs/require.js"
+		},
+		loaderOptions: {
+			paths: {
+				"globalize": "external/globalize/globalize",
+				"globalize/ja-JP": "external/globalize/globalize.culture.ja-JP",
+				"jquery": "external/jquery/jquery",
+				"jquery-simulate": "external/jquery-simulate/jquery.simulate",
+				"jshint": "external/jshint/jshint",
+				"lib": "tests/lib",
+				"qunit-assert-classes": "external/qunit-assert-classes/qunit-assert-classes",
+				// "qunit-assert-close": "external/qunit-assert-close/qunit-assert-close",
```

Comment:
I need to look into this issue more deeply, mapping was not working only for qunit-assert-close. Requirejs timed out always while loading it.